### ISSUE TITLE
loom: image (north) blit read: Bad address

### ISSUE THREAD

**Issue by dr-frmr** [urbit/urbit#6508]
**Describe the bug**

Pier unexpectedly shut down this morning:
<img width="704" alt="Screenshot 2023-04-25 at 12 38 54 PM" src="https://user-images.githubusercontent.com/93405247/234339716-ce419881-a61c-4fad-9581-4e98dbdf2c8e.png">

And will no longer boot, due to `loom: image (north) blit read: Bad address`.

My server has enough disk space and memory to allocate to this ship, and is running 2 other ships without issue.

**System:**
 - OS: Ubuntu 21.10
 - Vere: 2.1-8379b52

**Comment by joemfb** [comment]
See also #235. This is `EFAULT` on `read()` (of `north.bin`). How big are your checkpoint files? Have you tried with `--loom 33`?

**Comment by dr-frmr** [comment]
@joemfb my checkpoint files are 2.1G. 

I just booted with an 8G loom and it worked. But.. now I have to run it with an 8G loom forever, right?

**Comment by joemfb** [comment]
No. You can always switch to smaller loom, so long as your snapshot fits. That check was wrong, see urbit/vere#357.

I'm not sure why the 4GB loom didn't work for you.

**Comment by dr-frmr** [comment]
Ok -- I was just now able `|pack`, then boot at `--loom 32`. At no point was my loom close to 4G, I think it was under 2G at all times!

**Comment by vvisigoth** [comment]
I'm getting this error too, on attempting to do /.run next

When I try to run pack with --loom 33 it says can't map the 8G when the box clearly has room

![image](https://user-images.githubusercontent.com/1609955/236642369-547034af-bb75-40e6-8c85-57a5ae22f659.jpeg)

![image](https://user-images.githubusercontent.com/1609955/236642374-98005a55-aee1-46eb-b410-7ca717695dc4.jpeg)

**Comment by vvisigoth** [comment]
Jiggled the commands and it worked.
